datalad/distributed/ora_remote.py

```diff
@@ -267,6 +269,8 @@ def __init__(self, host):
                 break
         # TODO: Same for stderr?
 
+        self.buffer_size = buffer_size if buffer_size else 65536
```

Comment:
64k constant is defined twice in this file. Would make sense to use a single symbol.